node_modules/gh-users/tests/test-users.js

```diff
@@ -522,4 +523,137 @@ describe('Users', function() {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2014, 1, false, function(simon) {
+
+                UsersTestsUtil.assertChangePassword(simon.client, simon.profile.id, 'a new password', simon.client.options.password, function(newClient) {
+                    return callback();
+                });
+            });
+        });
+
+        /**
+         * Test that verifies validation when changing a user's password
+         */
+        it('verify validation when changing a user\'s password', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2014, 1, false, function(simon) {
+
+                var newPassword = 'new password';
+                var oldPassword = simon.client.options.password;
+
+                // Invalid user id
+                UsersTestsUtil.assertChangePasswordFails(simon.client, 'Not a number', newPassword, oldPassword, 400, function() {
+                    // Unknown user id
+                    UsersTestsUtil.assertChangePasswordFails(simon.client, -1, newPassword, oldPassword, 404, function() {
+
+                        // Missing new password
+                        UsersTestsUtil.assertChangePasswordFails(simon.client, simon.profile.id, null, oldPassword, 400, function() {
+
+                            // Too short new password
+                            UsersTestsUtil.assertChangePasswordFails(simon.client, simon.profile.id, 's', oldPassword, 400, function() {
+
+                                // Missing old password
+                                UsersTestsUtil.assertChangePasswordFails(simon.client, simon.profile.id, newPassword, null, 400, function() {
+
+                                    // Too short old password
+                                    UsersTestsUtil.assertChangePasswordFails(simon.client, simon.profile.id, newPassword, 's', 400, function() {
+
+                                        // Verify administrators don't need to provide an old password
+                                        UsersTestsUtil.assertChangePassword(global.tests.admins.cam2014.client, simon.profile.id, 'a new password', null, function(newClient) {
+                                            return callback();
+                                        });
+                                    });
+                                });
+                            });
+                        });
+                    });
+                });
+            });
+        });
+
+        /**
+         * Test that verifies authorization when changing a user's password
+         */
+        it('verify authorization when changing a user\'s password', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2014, 2, false, function(simon, nico) {
+
+                var newPassword = 'new password';
+                var oldPassword = simon.client.options.password;
+
+                // Anonymous users can't change other users their password
```

Comment:
I don't think it's super important to change this because I know exactly what it's saying, but I think I'll keep nagging you on this grammar structure any way :)

`// Anonymous users can't change other user passwords`

:sunglasses: